src/style.css

```diff
@@ -1,15 +1,8 @@
-fieldset {
-  margin-bottom: 20px;
-}
-
-input {
+.field_wrapper {
```

Comment:
following the BEM naming convention this name should have double underscore mark `field__wrapper`